notebooks/data-sources/github/thoth_PR_data_extraction.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    s3.upload_file(Filename="thoth_PR_data.csv", Bucket=s3_bucket, Key="thoth_PR_data.csv")
</pre><p>It seems like the definition of this <code>s3</code>  boto3.client object used here doesn't exist in the notebook. Was it accidentally deleted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/498/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Oops. Yes it is accidental. Thanks for pointing out. I will add it.